build.gradle.kts

```diff
@@ -342,7 +342,7 @@ tasks {
 		enabled = !experimentalBuild
 		reports {
 			xml.required.set(true)
-			xml.outputLocation.set(file("${layout.buildDirectory}/reports/jacoco/report.xml"))
```

Comment:
This is already fixed on `main`:

https://github.com/junit-pioneer/junit-pioneer/blob/main/build.gradle.kts#L339

Reply:
Will pull.